examples/farmer/farmer_rho_demo.py

```diff
@@ -72,14 +76,16 @@ def main():
 
     num_scen = cfg.num_scens
     crops_multiplier = cfg.crops_mult
-    rho_setter = None
+    rho_setter = None  # non-grad rho setter?
 
     if cfg.default_rho is None and rho_setter is None:
         raise RuntimeError("No rho_setter so a default must be specified via --default-rho")
 
     if cfg.use_norm_rho_converger:
         if not cfg.use_norm_rho_updater:
             raise RuntimeError("--use-norm-rho-converger requires --use-norm-rho-updater")
+        elif cfg.grad_rho_setter:
+            raise RuntimeError("You cannot have--use-norm-rho-converger and --grad-rho-setter")            
```

Comment:
Largely unproductive whining, which I should put in an issue - we should be consistent and have "--norm-rho-converger" without the "--use" prefix.

Reply:
I agree